### DISCUSSION TITLE
Local network DNS lookups to home.arpa

### DISCUSSION THREAD

**Discussion by notDavid** [mullvad/mullvadvpn-app#9123] [Feature requests & Ideas]
## Issue report

Operating system: macOS 13.3.1 (a)

App version: 2023.3

### Issue description

Hi there, so my question is, with 'Local Network Sharing' enabled, please consider forwarding dns lookups for the [local home.arpa domain](https://www.ctrl.blog/entry/homenet-domain-name.html) to the local dns server.

Thank you!

<details>
  <summary>example sent to Mullvad dns with vpn up</summary>

```log
dig mylocalpc.home.arpa

; <<>> DiG 9.10.6 <<>> mylocalpc.home.arpa
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 44171
;; flags: qr aa rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;mylocalpc.home.arpa.		IN	A

;; AUTHORITY SECTION:
HOME.ARPA.		86400	IN	SOA	HOME.ARPA. . 0 28800 7200 604800 86400

;; Query time: 39 msec
;; SERVER: 100.64.0.31#53(100.64.0.31)
;; WHEN: Sat May 13 10:26:10 CEST 2023
;; MSG SIZE  rcvd: 92
```
</details>

**Comment by ghost** [comment]
This would also be useful in the Linux version. I'd assume the Windows version as well.

In the meantime, `Local network sharing` does allow mDNS to work; so Avahi, and whatever is offered on other OSs, are solid alternatives.
